docs/_static/custom.css

```diff
@@ -55,7 +55,9 @@ code.xref.docutils.literal {
 }
 
 div.viewcode-block:target {
-    background: inherit;
+    background:#dadada;
```

Comment:
Could we add a space before `#dadada` for consistency with the rest of the stylesheet?

Reply:
Fixed. VSCode plops in a color picker to the left so I didn't see the missed space hah